internal/bminventory/inventory.go

```diff
@@ -767,6 +767,17 @@ func (b *bareMetalInventory) createAndUploadNodeIgnition(ctx context.Context, cl
 		return errors.Errorf("Failed to create ignition string for cluster %s", cluster.ID)
 	}
 
+	// Apply overrides to the ignition
+	if host.IgnitionConfigOverrides != "" {
+		var res string
+		res, err = ignition.MergeIgnitionConfig(ignitionBytes, []byte(host.IgnitionConfigOverrides))
+		if err != nil {
+			return errors.Errorf("Failed to apply ignition override for host %s", host.ID)
```

Comment:
return the actual error
```suggestion
			return errors.Wrap(err, "Failed to apply ignition override for host %s", host.ID)
```

Reply:
ACK